util/rpy2rotmat.m

```diff
@@ -99,4 +99,135 @@
   drotmat_val(21) = -sin_p*cos_r;
   drotmat = sparse(drotmat_row,drotmat_col,drotmat_val,9,3);
 end
+if(nargout>2)
+  rr_idx = 1;
+  rp_idx = 2;
+  ry_idx = 3;
+  pr_idx = 4;
+  pp_idx = 5;
+  py_idx = 6;
+  yr_idx = 7;
+  yp_idx = 8;
+  yy_idx = 9;
+  
+  ddR_row = zeros(53,1);
+  ddR_col = zeros(53,1);
+  ddR_val = zeros(53,1);
+  ddR_nnz = 1;
+  
+  [ddR_row,ddR_col,ddR_val,ddR_nnz] = addEntry_ddR(1,yy_idx,-cos_y*cos_p,ddR_row,ddR_col,ddR_val,ddR_nnz);
```

Comment:
isn't the normal approach for this to make an all-zeros sparse matrix, then just assign the elements of it with `ddrotmat(1,yy_idx) = val`.  that would be a lot cleaner.